server/resolvers/testPlanReportsResolver.js

```diff
@@ -10,13 +10,14 @@ const {
 
 const testPlanReportsResolver = async (
     _,
-    { statuses, testPlanVersionId, atId },
+    { statuses, testPlanVersionId, testPlanVersionIds, atId },
     context,
     info
 ) => {
     const where = {};
     if (statuses) where.status = statuses;
     if (testPlanVersionId) where.testPlanVersionId = testPlanVersionId;
+    if (testPlanVersionIds) where.testPlanVersionId = testPlanVersionIds;
```

Comment:
Good question! But this is fine as it is. The `testPlanVersionId` here is referring to the column on the `TestPlanReport` table and what `Sequelize` will do here when the `WHERE` clause is used with an array instead of a single value, it will infer the DB statement as being:

> `WHERE ... AND "TestPlanReport"."testPlanVersionId" IN ('1090', '1762') ...`, given that `testPlanVersionIds` is `['1090', '1762']`

instead of:

> `WHERE ... AND "TestPlanReport"."testPlanVersionId" = '1090' ...`